bl/evaluation/printer.go

```diff
@@ -287,45 +287,45 @@ func parseEvaluationResultsToSummary(results *EvaluationResults, evaluationSumma
 	return *summary
 }
 
-func (mapper FileNameRuleMapper) MarshalXML(e *xml.Encoder, start xml.StartElement) error {
-	if len(mapper) == 0 {
-		return nil
-	}
-
-	tokens := []xml.Token{start}
-	fileXMLName := xml.Name{Space: "", Local: "File"}
-	filenameXMLName := xml.Name{Space: "", Local: "filename"}
-
-	// Iterate over mapper and create XML tokens for all entries
-	for filePath, encapsulatedRule := range mapper {
-		keys := make([]int, 0)
-		for i := range encapsulatedRule {
-			keys = append(keys, i)
-		}
-		sort.Ints(keys)
-
-		for _, key := range keys { // Since rule is encapsulated by its id (int), we don't add is a tag
-			rule := encapsulatedRule[key]
-			startToken := xml.StartElement{Name: fileXMLName, Attr: []xml.Attr{{Name: filenameXMLName, Value: filePath}}}
-			endToken := xml.EndElement{Name: fileXMLName}
-			tokens = append(tokens, startToken, rule, endToken)
-		}
-	}
-
-	tokens = append(tokens, xml.EndElement{Name: start.Name})
-
-	for _, t := range tokens {
-		var err error
-		switch t.(type) {
-		default:
-			err = e.EncodeToken(t)
-		case *Rule:
-			err = e.EncodeElement(t, xml.StartElement{Name: xml.Name{Space: "", Local: "Rule"}})
-		}
-		if err != nil {
-			return err
-		}
-	}
-
-	return e.Flush()
-}
+//func (mapper FileNameRuleMapper) MarshalXML(e *xml.Encoder, start xml.StartElement) error {
+//	if len(mapper) == 0 {
+//		return nil
+//	}
+//
+//	tokens := []xml.Token{start}
+//	fileXMLName := xml.Name{Space: "", Local: "File"}
+//	filenameXMLName := xml.Name{Space: "", Local: "filename"}
+//
+//	// Iterate over mapper and create XML tokens for all entries
+//	for filePath, encapsulatedRule := range mapper {
+//		keys := make([]int, 0)
+//		for i := range encapsulatedRule {
+//			keys = append(keys, i)
+//		}
+//		sort.Ints(keys)
+//
+//		for _, key := range keys { // Since rule is encapsulated by its id (int), we don't add is a tag
+//			rule := encapsulatedRule[key]
```

Comment:
Not sure if we need this code - ATM it's unused. It can work with local evaluation.